### PR TITLE
lots of misc doc updates

### PR DESCRIPTION
**Pull request by wmorgan**
Lots of random updates and some content shuffling. Might be easiest to review these together in person.

Signed-off-by: William Morgan <william@buoyant.io>

### CODE REVIEW

File: linkerd.io/content/2/getting-started/_index.md [hunk 1]
```diff
@@ -271,13 +277,12 @@ Grafana icon in the overview page.
...
-of these:
+Congratulations, you're now a Linkerd user! Here are some suggested next steps:
 
-- [Debug a service](/2/debugging-an-app/)
-- [Add Linkerd to your service](/2/adding-your-service/)
-- [Learn more](/2/reference/architecture/) about Linkerd's architecture
-- Hop into the #linkerd2 channel on
-  [the Linkerd Slack](https://slack.linkerd.io)
+- Use Linkerd to [debug the errors in *emojivoto*](/2/debugging-an-app/)
+- [Add own service](/2/adding-your-service/) to Linkerd without downtime
```
Comment (cpretzer) [inline linkerd.io/content/2/getting-started/_index.md]: "Add your own service" is consistent with the tone of the document

File: linkerd.io/content/2/tasks/modifying-proxy-log-level.md [hunk 2]
```diff
@@ -1,12 +1,26 @@
...
-`kubectl edit` to change the `LINKERD2_PROXY_LOG` environment variable of the
-resource. For auto-injected proxies, the `config.linkerd.io/proxy-log-level`
-annotation can also be used to override the existing log level.
+Emitting logs is an expensive operation for a network proxy, and by default,
+the Linkerd data plane proxies are configured to only log exceptional events.
+However, sometimes it is useful to increase the verbosity of proxy logs to
+assist with diagnosing proxy behavior. Happily, Linkerd allows you to modify
+these logs dynamically.
+
+The log level a Linkerd proxt can be modified on the fly by setting the
```
Comment (grampelberg) [inline linkerd.io/content/2/tasks/modifying-proxy-log-level.md]: proxt -> proxy

File: linkerd.io/content/2/tasks/modifying-proxy-log-level.md [hunk 3]
```diff
@@ -1,12 +1,26 @@
...
+`config.linkerd.io/proxy-log-level` annotation on the pod.  Alternatively, the
+log level can also be changed by modifying the the `LINKERD2_PROXY_LOG`
```
Comment (grampelberg) [inline linkerd.io/content/2/tasks/modifying-proxy-log-level.md]: I don't think this'll actually work anymore, unless they're doing `inject --manual`. It is probably worth just explaining the annotation and leaving it at that.

File: linkerd.io/content/2/tasks/securing-your-service.md [hunk 4]
```diff
@@ -3,23 +3,22 @@ title = "Securing Your Service"
 description = "Linkerd encrypts your service's traffic by default."
 +++
 
-By default, Linkerd automatically enables mutual Transport Layer Security (mTLS)
-for all HTTP-based communication between meshed pods, by establishing and
-authenticating secure, private TLS connections between Linkerd proxies. [Add
-your services](/2/tasks/adding-your-service/) and Linkerd will take care of the
-rest.
-
-{{< note >}}
```
Comment (grampelberg) [inline linkerd.io/content/2/tasks/securing-your-service.md]: I know you moved this one over to the features section, but it feels important here as this is primarily a debugging doc for mTLS.

File: linkerd.io/content/2/tasks/securing-your-service.md [hunk 5]
```diff
@@ -56,50 +58,61 @@ rsp id=0:7 proxy=in  src=10.138.15.206:51558 dst=10.4.0.18:9998 tls=not_provided
...
 
-To watch the actual wire traffic, Linkerd includes a [debug
-sidecar](/2/tasks/using-the-debug-container/) that comes with a selection of
-commands that make it easier to verify and debug the service mesh itself. To
-take a look at this with [emojivoto](/2/getting-started/), get the demo started
-and add the debug sidecar by running:
+## Validating mTLS with tshark
+
+The final way to validate mTLS is to look at raw network traffic. (Note that
+this approach requires full access to the Kubernetes node.)
```
Comment (grampelberg) [inline linkerd.io/content/2/tasks/securing-your-service.md]: It doesn't require any access to the k8s node?

File: linkerd.io/content/2/tasks/uninstall.md [hunk 6]
```diff
@@ -1,11 +1,21 @@
...
+injection annotations](/2/features/proxy-injection/) and roll the deployments.
+When Kubernetes recreates the pods, they will not have the Linkerd data plane
+attached.
+
+One way to do this is to use the `linkerd uninject` command to rewrite a
+Kubernetes manifest. For example, to do this across all namespaces in one fell
+swoop:
 
 ```bash
 kubectl get --all-namespaces daemonset,deploy,job,statefulset \
```
Comment (grampelberg) [inline linkerd.io/content/2/tasks/uninstall.md]: @ihcsim @alpeb as we're doing auto-inject for pretty much everything now, this won't work anymore. You wouldn't happen to know of an updated command?

Comment (ihcsim) [reply linkerd.io/content/2/tasks/uninstall.md]: AFAICT, `uninject` doesn't work with the current inject workflow. I created https://github.com/linkerd/linkerd2/issues/3156 as part of the 2.5 backlog to fix this.

Comment (grampelberg) [reply linkerd.io/content/2/tasks/uninstall.md]: Could we do a `kubectl patch` in the meantime?

File: linkerd.io/content/2/tasks/securing-your-service.md (hunk 4 above)
Comment (wmorgan) [reply linkerd.io/content/2/tasks/securing-your-service.md]: Done!

File: linkerd.io/content/2/tasks/uninstall.md (hunk 6 above)
Comment (wmorgan) [reply linkerd.io/content/2/tasks/uninstall.md]: I just removed the `uninject` oneliner... if we find a fancy way of doing this in the future we can put it back in.

**Comment by wmorgan** [comment]
@grampelberg PTAL when you get a chance

**Pull request merged**